vite.md

```diff
@@ -410,6 +411,25 @@ https://cdn.example.com/build/assets/app.9dce8d17.js
 
 Remember that [absolute URLs are not re-written by Vite](#url-processing), so they will not be prefixed.
 
+<a name="override-build-path"></a>
+## Override Build Path
```

Comment:
Rather than add another top-level heading, I think this would be better in the "Configuring Vite" section.

Reply:
Please read my reply below